.travis.yml

```diff
@@ -54,10 +55,9 @@ matrix:
 
   - node_js: 8
     env:
-      - TEST_SUITE=closure
-      - CLOSURE=1
+      - TEST_SUITE=typescript
     script:
-      - if has_testable_files; then npm run test:closure; else log_untestable_files; fi
+      - if has_testable_files; then npm run test:typescript; else log_untestable_files; fi
```

Comment:
This npm script does not exist. Should we remove this? We're already testing building the TS/JS in the build task.